src/components/Playbar/index.tsx

```diff
@@ -140,16 +142,47 @@ export function PlaybarUI(props: PlaybarUIProps): ReactElement {
                         className="flex flex-col h-[30px] justify-center flex-shrink-0 cursor-pointer w-[72px] ml-1 group"
                         onClick={onClickCalendar}
                     >
-                        <div className="flex items-end h-[12px] mt-[1px]">
-                            <span className="select-none leading-none text-sm text-blue-600 font-bold uppercase group-hover:text-blue-800 group-active:text-blue-900">
-                                {eventMetaData?.quote?.localTicker || 'Instrument'}
-                            </span>
-                            <span className="select-none truncate leading-none ml-1 mb-[1px] text-xxs uppercase tracking-widest text-gray-400 group-hover:text-gray-600 group-active:text-gray-800">
-                                {eventMetaData?.quote?.exchange?.shortName || 'Exchange'}
-                            </span>
-                        </div>
+                        {isCustom && eventMetaData?.title ? (
+                            <Tooltip
+                                className="flex items-end h-[12px] mt-[1px]"
+                                content={
+                                    <div className="bg-black bg-opacity-80 max-w-[300px] px-1.5 py-0.5 rounded text-sm text-white dark:bg-bluegray-4 dark:text-bluegray-7">
+                                        {prettyLineBreak(eventMetaData.title)}
+                                    </div>
+                                }
+                                grow="up-right"
+                                hideOnDocumentScroll
+                                openOn="hover"
+                                position="top-left"
+                                yOffset={30}
+                            >
+                                <span className="font-bold leading-none overflow-hidden select-none text-blue-600 text-ellipsis text-sm whitespace-nowrap group-active:text-blue-900 group-hover:text-blue-800">
+                                    {eventMetaData?.quote?.localTicker || eventMetaData.title}
+                                </span>
+                            </Tooltip>
+                        ) : (
+                            <div className="flex items-end h-[12px] mt-[1px]">
+                                <span
+                                    className={classNames(
+                                        'select-none leading-none text-sm text-blue-600 font-bold overflow-hidden text-ellipsis whitespace-nowrap group-hover:text-blue-800 group-active:text-blue-900',
+                                        {
+                                            uppercase: !isCustom,
+                                        }
+                                    )}
+                                >
+                                    {eventMetaData?.quote?.localTicker || eventMetaData?.title || 'Instrument'}
+                                </span>
+                                {!isCustom && (
```

Comment:
Ditto as the comment above, no need for `!isCustom` here, but already fixed so this will be resolved